### DISCUSSION TITLE
Verifying Conversion of Relative Path URI's to Link API

### DISCUSSION THREAD

**Discussion by jadamconnor** [gatsbyjs/gatsby#34195] [Help]
I'm wondering if there's a way to verify that URI's used in Wordpress post content are indeed using Gatsby's Link API as stated in this feature page: 

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/features/gatsby-link.md

I have a large number of internal links that our content editors incorrectly had opening in new tabs. I went through the database and removed all the target attributes from those anchor tags and now each of the internal links do, in fact, show relative paths in the href attribute value. But according to Gatsby docs the Link API preloads these pages, and I'm not noticing particularly fast navigation here.

TLDR: Just wondering if there's a way to verify that anchor elements are in fact being treated as Gatby internal Links by the plug-in.
